test/library/standard/Types/isNothing-proc.chpl

```diff
@@ -0,0 +1,32 @@
+use Types;
+
+proc isNothing(type t)  param return isNothingType(t);
+proc isNothing(e)   param return isNothingType(e.type);
+
+
+proc myassert(param isNothing){
+writeln(isNothing);
```

Comment:
let's have this function instead take in a `type t` argument and then you can print out both the type and the result of `isNothing` like this:

``` chapel
proc myassert(type t) {
  var dummyVar: t;
  writeln(t:string, " ", isNothing(t), " ", isNothing(dummyVar));
}
```